client/settings/deposits/index.js

```diff
@@ -162,6 +180,12 @@ const DepositsSchedule = () => {
 					href="https://woocommerce.com/document/payments/faq/deposit-schedule/"
 					target="_blank"
 					rel="external noreferrer noopener"
+					onClick={ () =>
+						wcpayTracks.recordEvent(
+							'wcpay_deposits_schedule_disabled_help',
```

Comment:
Quick feedback on the event naming (assuming this is still in progress):

- Use a consistent prefix for the event source, i.e. where events are triggered. Suggest `wcpay_settings_deposits_`.
- Name the action similar to the UI element – we are tracking the user clicking or viewing something. 
- Use an action suffix, e.g. `view` or `click`.

So for this example, I'd suggest something like: `wcpay_settings_deposits_new_account_notice_help_click`, similar to proposed [`wcpay_overview_deposits_new_account_notice_docs_click`](https://github.com/Automattic/woocommerce-payments/issues/5881#issuecomment-1530793008).

Though … we might not need this event yet (I didn't add that event to overview page).

Reply:
> Quick feedback on the event naming (assuming this is still in progress)
It was still in progress and has now been updated.